api/tests/emails/offer_creation_test.py

```diff
@@ -55,11 +57,16 @@ def test_with_physical_offer(self, isEducationalOffer):
         assert f"Lien vers l'offre dans la Webapp :" f" {settings.WEBAPP_V2_URL}/offre/{offer.id}" in webapp_offer_link
 
         pro_offer_link = str(parsed_email.find("p", {"id": "pro_offer_link"}))
-        pro_offer_type = "individuel" if isEducationalOffer is False else "collectif"
-        assert (
-            f"Lien vers l'offre dans le portail PRO :"
-            f" http://localhost:3001/offre/{humanize(offer.id)}/{pro_offer_type}/edition" in pro_offer_link
-        )
+        if isEducationalOffer is True:
```

Comment:
(style) Au passage, on peut supprimer ce "is True/False" : 

```suggestion
        if isEducationalOffer:
```